packages/neon/neon/lib/src/models/account.dart

```diff
@@ -62,8 +68,13 @@ class Account implements Credentials {
   @override
   int get hashCode => serverURL.hashCode + username.hashCode;
 
+  /// An authenticated api client.
```

Comment:
```suggestion
  /// An authenticated API client.
```

Reply:
Should we add `api` in lowercase to the cspell config?
It's not the first time you request this change.

Reply:
Good idea. Please check if there are valid uses of the lowercase version first

Reply:
There are many within dynamite and as part of the word `openapi`.
There is also the linting rule `public_member_api_docs`.

We probably can't enforce this :(

Reply:
Too bad, but please fix it here